pkg/util/cmdutil/spinner.go

```diff
@@ -35,7 +35,7 @@ func NewSpinnerAndTicker(prefix string, ttyFrames []string, timesPerSecond time.
 		}
 	}
 
-	if InteractiveTerminal() {
+	if Interactive() {
```

Comment:
this is the only functional change.  Note: we could potentially be more fine grained here and only do this for circleci.  However, i don't get the value here. We're already doing the line-by-line non-interactive output.  This just makes it so that we do:

`@ Updating ...............`

Instead of trying to do `@ Updated ... %spinner%`

On a CI machine you'll still see it making progress every 20 seconds, and there won't be any need to worry about any particular term issues of each system.